### ISSUE TITLE
Display Actions do nothing and provide no feedback on unlicensed displays

### ISSUE THREAD

**Issue by PatrickA-RiseVision** [Rise-Vision/rise-vision-apps#2519]
### What is the expected behavior?
There is feedback when you perform a display action.  Either a modal confirming the action took place, or a modal prompting you to license your displays.

### What is the actual behavior?
When a display action is performed from within the display settings on an unlicensed display, nothing happens at all.

### What steps can be taken to replicate the problem?
As a user, click on an unlicensed display and attempt to restart it using Display Actions.

**Comment by ezequielc** [comment]
@PatrickA-RiseVision Are you referring to the greyed out actions below?
<img width="473" alt="Screen Shot 2021-04-09 at 2 37 13 PM" src="https://user-images.githubusercontent.com/7738553/114219619-49804880-9941-11eb-91eb-80893ce0ac9a.png">

If so, I believe that is intentional, those action are disabled and by clicking on them it will just dismiss the dropdown.
@jasoncaldwell Any feedback here? Thanks!

**Comment by jasoncaldwell** [comment]
@ezequielc I believe in @PatrickA-RiseVision's testing, the menu items weren't disabled. @PatrickA-RiseVision, can you confirm?

**Comment by PatrickA-RiseVision** [comment]
Correct, they aren`t greyed out.  https://drive.google.com/file/d/1J_NNvVI5nusjVNyXzzLTUDg-799544ES/view?usp=drivesdk

I guess it could be a caching issue if you can confirm that it works as intended for you @ezequielc

**Comment by ezequielc** [comment]
@PatrickA-RiseVision Yeah, it worked fine for me, but I will keep the defect open and take a better look at it in our cooldown. There may be some specific conditions causing it to not be disabled for you. Thanks

**Comment by alex-deaconu** [comment]
@PatrickA-RiseVision @jasoncaldwell @ezequielc I just want to clarify the logic for the Restart/Reboot options in that dropdown:
- The option is disabled exclusively if the `playerVersion` field is not present, ie. the Display is not installed or pending receiving the install information (separate issue).
- The option is enabled otherwise, and if clicked on an Unlicensed Display, will show a License warning.
- However, if you're a Rise Admin this license warning is bypassed, allowing you to send a Restart/Reboot command.

Now, there are a few variables at play here. First off, if the Display is offline, obviously nothing will happen. Also, I'm not really sure if the restart/reboot are disabled somewhere down the line (ie. Core, Messaging) for unlicensed displays, in which case a Rise Admin would not really be able to perform the action.

To summarize, @PatrickA-RiseVision if you were using your Rise Admin account you would be able to perform that action on the Display, while a regular User would receive the License warning. Wondering if you could confirm the circumstances for this issue so we can proceed.

Thanks!

**Comment by alex-deaconu** [comment]
@PatrickA-RiseVision wondering if you could get back to me on this. Thanks!

**Comment by PatrickA-RiseVision** [comment]
Sorry for the delay @alex-deaconu .  In this case, I was an admin at the time, so it sounds like this is expected behavior.

**Comment by alex-deaconu** [comment]
@PatrickA-RiseVision thanks for confirming! I will close the defect at this point since there are no actions to pursue. Thanks!
